docs/hadoop-catalog.md

```diff
@@ -76,6 +76,18 @@ In the meantime, you need to place the corresponding bundle jar [`gravitino-gcp-
 
 In the meantime, you need to place the corresponding bundle jar [`gravitino-aliyun-bundle-${version}.jar`](https://repo1.maven.org/maven2/org/apache/gravitino/aliyun-bundle/) in the directory `${GRAVITINO_HOME}/catalogs/hadoop/libs`.
 
+
+#### Azure Blob Storage fileset
+
+| Configuration item            | Description                                                                                                                                                                                                                                   | Default value   | Required                                  | Since version    |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|-------------------------------------------|------------------|
+| `filesystem-providers`        | The file system providers to add. Set it to `abfs` if it's a Azure block storage fileset, or a comma separated string that contains `abfs` like `oss,abfs,s3` to support multiple kinds of fileset including `abfs`.                          | (none)          | Yes                                       | 0.8.0-incubating |
+| `default-filesystem-provider` | The name default filesystem providers of this Hadoop catalog if users do not specify the scheme in the URI. Default value is `builtin-local`, for Azure block store, if we set this value, we can omit the prefix 'abfss://' in the location. | `builtin-local` | No                                        | 0.8.0-incubating |
```

Comment:
"blob storage" or "blob store", I think you should follow the official name from Azure.